polyhost/gui/fontpack_extend_dialog.py

```
@@ -0,0 +1,536 @@
+            self._list.setCurrentRow(0)
+        v.addWidget(self._list, 1)
+
+        btns = QHBoxLayout()
+        self._all_btn = QPushButton("Download all")
+        self._all_btn.setToolTip("Fetch every Noto font in the list into the cache")
+        self._all_btn.clicked.connect(self._download_all)
+        self._all_btn.setEnabled(bool(self._fonts))
+        self._dl_btn = QPushButton("Download / Use selected")
+        self._dl_btn.clicked.connect(self._download)
+        self._dl_btn.setEnabled(bool(self._fonts))
+        cancel = QPushButton("Close"); cancel.clicked.connect(self.reject)
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)